data_model.md

```diff
@@ -226,10 +229,37 @@ for more detailed explanation of forecast parameters):
 Additional parameters, such as model configuration parameters, may be
 specified under the “extra parameters” key.
 
-
 Each Forecast has data associated with it. The format of this data is
 found in the Data Format section below
 
+#### Probabilistic forecasts
+{: .anchor}
+
+The Solar Forecast Arbiter supports the specification of a full
+probabilistic forecast in terms of a cumulative distribution function
+(CDF). The metadata for a probabilistic forecast container is the same
+as for a [single-valued forecast](#single-valued-forecasts) with the
+addition of two attributes:
+
+-   *Axis* - The axis on which the CDF is specified. *x* corresponds to
```

Comment:
`*Axis* - values of the variable being forecast, i.e., the *x* axis of the CDF.`
`*Points* - quantile values in the range from 0 to 1, i.e., the *y* axis of the CDF.`